nevergrad/parametrization/choice.py

```diff
@@ -156,6 +159,7 @@ def __init__(
         rep = 1 if repetitions is None else repetitions
         super().__init__(choices=lchoices, repetitions=repetitions,
                          weights=Array(shape=(rep, len(lchoices)), mutable_sigma=False))
+        self.weights.heritage[BaseChoice.ChoiceTag] = BaseChoice.ChoiceTag(self.__class__, len(lchoices))
```

Comment:
@teytaud this is a hack so we can identified that an Array is used as weights for a choice